src/extensions/default/InlineBorderRadiusEditor/BorderRadiusEditor.js

```diff
@@ -0,0 +1,818 @@
+     * @type {?string}
+     */
+    BorderRadiusEditor.prototype._redoValues = null;
+
+    /**
+     * Initial value the BoxShadow picker was opened with
+     * @type {!string}
+     */
+    BorderRadiusEditor.prototype._originalValues = null;
+
+
+    /** Returns the root DOM node of the BoxShadowPicker UI */
+    BorderRadiusEditor.prototype.getRootElement = function () {
+        return this.$element;
+    };
+
+    BorderRadiusEditor.prototype.getAllSliders = function() {
+        var sliders = {
+            tr : this.$trslider,
+            tl : this.$tlslider,
+            br : this.$brslider,
+            bl : this.$blslider,
+            all : this.$allCornerSlider
+        };
+        return sliders;
+    };
+
+    BorderRadiusEditor.prototype.getAllRadios = function() {
+        var radios = {
+            tr : this.$tlradio,
+            tl : this.$trradio,
+            br : this.$brradio,
+            bl : this.$blradio,
+            all : this.$allradio
+        };
+        return radios;
+    };
+
+    BorderRadiusEditor.prototype.setValues = function(values) {
+        var result = getIndividualValues(values.replaceAll(" ","").replaceAll(";",""));
+        if(!result){
+            return;
+        }
+
+
+        //var result = values.replace(' ','').replace(";","").split("px");
+        var finalValue = "";
+        //var count=0;
+        /*for(var i = 0; i < result.length; i++){
+            if(!isNaN(parseFloat(result[i]))){
+                result[i] = parseFloat(result[i])+"px";
+                finalValue += result[i];
+                count++;
+            }
+        }*/
+
+        for(var i = 0; i<result.length;i++){
+            finalValue += (result[i].num+result[i].unit);
+        }
+
+
+        //this._allCorners=(count===1||count===0);
+        this._allCorners = (result.length === 1);
+        this._values = finalValue;
+        this.individualValuesWithUnit = result;
+        /*if(count===0){
+            this._values = DEFAULT_BORDER_RADIUS_VALUE + "px";
+        }*/
+        this._setInputValues(true);
+        this._commitChanges(values);
+    };
+
+    BorderRadiusEditor.prototype._setInputValues = function(setFromString) {
+        var values = this.individualValuesWithUnit;
+        //this._values.split("px");
+        //var tl,tr,bl,br,all;
+            if(!this._allCorners){
+                if(values.length===1 && (this._init || setFromString)){
+
+                    this._tr = parseFloat(values[0].num);
+                    this._tl = parseFloat(values[0].num);
+                    this._br = parseFloat(values[0].num);
+                    this._bl = parseFloat(values[0].num);
+                    this._tlUnit=values[0].unit;
+                    this._trUnit=values[0].unit;
+                    this._brUnit=values[0].unit;
+                    this._blUnit= values[0].unit;
+                    this._allUnit=this._allUnit || "px";
+                    this._all = this._all || 0;
+
+                }
```

Comment:
Same deal here and everywhere, `} else {`.  Always put your braces on the same line.